cmd/namespace/create.go

```diff
@@ -84,13 +86,19 @@ func (nc *NamespaceCommand) Create(ctx context.Context, opts *CreateOptions) err
 	ctxOptions := &contextCMD.ContextOptions{
 		IsCtxCommand: opts.Show,
 		IsOkteto:     true,
-		Save:         true,
-		Show:         false,
 		Token:        okteto.Context().Token,
 		Namespace:    oktetoNS,
 		Context:      okteto.Context().Name,
 	}
 
+	if opts.SetCurrent == true {
```

Comment:
@jLopezbarb Would you mind taking a look, how can i improve this piece of code?

Reply:
Right now there is no other way of doing this... So I guess this is the correct approach